### DISCUSSION TITLE
Single-use debit cards

### DISCUSSION THREAD

**Discussion by AedenThomas** [simple-login/app#482] [Feature Requests]
Sites like DoNotPay, Privacy.com offer this. Using a single-use card will help in many things for example taking a trial without disclosing our personal Banking Information. Hope SimpleLogin can bring this too in the future

**Comment by nguyenkims** [comment]
Hi we rely on Paddle for handling payments and it sometimes refuses single-use cards. SimpleLogin also supports cryptocurrency for as a more anonymous option. In the future, I hope we can add other payment providers like Stripe to reduce the risk of a card being refused.

**Reply by AedenThomas** [reply]
I'm sorry. I guess I wasn't clear. What I meant is a feature in SimpleLogin to generate Single Use debit cards which can be used in sites that require a card for verification. Sites like DoNotPay, Privacy.com offer this and it would be cool if SimpleLogin also did.

**Reply by nguyenkims** [reply]
Oh I see, being able to create single-use bank card is part of SimpleLogin's long term plan indeed!

**Reply by fcercwer3** [reply]
When do you want to launch it? Have a timetable for it?

**Reply by nguyenkims** [reply]
We plan to launch this feature in alpha at the beginning of 2023.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Hi @nguyenkims is it in early alpha as we speak?

**Reply by nguyenkims** [reply]
Unfortunately not. We have spent a good chunk of 2023 to scale SL which has much more users than before since we joined Proton. We hope to be able to study this subject soon but as of now, I can't speculate on the ETA as this depends on other ongoing projects.
